src/runtime/runtime.js

```diff
@@ -1168,6 +1172,15 @@ export class ConfiguredRuntime {
       showcaseEventToAnalyticsEvents(entitlement.entitlement) || [];
     const params = new EventParams();
     params.setIsUserRegistered(entitlement.isUserRegistered);
+    if (entitlement.subscriptionTimestamp) {
+      params.setSubscriptionTimestamp(
```

Comment:
Would it be possible to add test coverage for this line?

Reply:
New test can cover this line, thanks!

Reply:
Sweet!